crm_compassion/models/interaction_resume.py

```diff
@@ -19,13 +19,14 @@ class InteractionResume(models.TransientModel):
     partner_id = fields.Many2one("res.partner", "Partner")
     email = fields.Char()
     communication_type = fields.Selection(
-        [("Paper", "Paper"), ("Phone", "Phone"), ("SMS", "SMS"), ("Email", "Email")]
+        [("Paper", "Paper"), ("Phone", "Phone"), ("SMS", "SMS"), ("Email", "Email"), ("Other", "Other")]
     )
     direction = fields.Selection([("in", "Incoming"), ("out", "Outgoing"), ])
     # Used to display icons in tree view
     state = fields.Selection(related="direction")
     communication_date = fields.Datetime()
     subject = fields.Text()
+    other_type = fields.Text()
```

Comment:
To be consistent with the type used in the wizard.
```suggestion
    other_type = fields.Char()
```